src/ydata_synthetic/preprocessing/timeseries/utils.py

```diff
@@ -4,7 +4,7 @@
 import numpy as np
```

Comment:
Enabling inverse_transform of the timeseries scaler (processor needs to be returned here)